src/contracts/mainnet/AaveV3EthCBETHSupplyCapsPayload.sol

```diff
@@ -0,0 +1,25 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.10;
```

Comment:
```suggestion
pragma solidity ^0.8.16;
```

const reassignment like `  address public constant CBETH = AaveV3EthereumAssets.cbETH_UNDERLYING;
` need `16+` - i know it's not done consistently, will fix globally at some point.